src/improvements/template/OPCMUpgradeV200.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {OPCMBaseTask} from "../tasks/OPCMBaseTask.sol";
+import {AddressRegistry} from "src/improvements/AddressRegistry.sol";
+import {
+    IOPContractsManager,
+    ISystemConfig,
+    IProxyAdmin
+} from "@eth-optimism-bedrock/interfaces/L1/IOPContractsManager.sol";
+import {Claim} from "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+
+/// @notice This is an example of implementing OPCMBaseTask to perform an upgrade via the OPCM contract.
```

Comment:
You can change these comments to be more specific to this actual template i.e. OPCM Version 2.0.0

I'm also thinking that this task is specific for sepolia only because the constant `OPCM` is on sepolia (`0x81395Ec06F830a3B83FE64917893193380a58d11`).

I'm thinking we might want this template but then a way to either supply the OPCM address via config or have 3 templates in total. OPCMv200 Base, OPCMv200 Sepolia, OPCMv200 Mainnet.

Reply:
This looks good, reading opcm address from config makes sense to me.